nengo_gui/static/components/netgraph.js

```diff
@@ -307,9 +307,6 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var item = this.svg_objects[data.uid];
         item.set_label(data.name);
 
-        var item_mini = this.minimap_objects[data.uid];
-        item_mini.set_label(data.name);
-
```

Comment:
This was something I corrected while debugging. It was probably a side-effect of the above error. This label was never actually used for anything.